docs/events/add-manage-notes.asciidoc

```diff
@@ -0,0 +1,78 @@
+. Enter a note, then click **Add note**.
++
+[role="screenshot"]
+image::images/create-new-note.png[Creating a new note]
++
+In the Alerts or Events tables, a notification dot displays on alerts and events with notes. Hover over the **Add note** icon (image:images/create-note-icon.png[Add note,15,15]) to see how many notes are attached to the alert or event.
++
+[role="screenshot"]
+image::images/notes-notification.png[Notes notification marker on Alerts page]
+
+To add a note from the alert or event details flyout:
+
+. Go to the Alerts or Events tables, and open the details flyout of the alert or event.
+. Do one of the following:
+.. In the alert details flyout, click **Add note**. The **Notes** tab in the left panel displays.
++
+[role="screenshot"]
+image::images/add-new-note-flyout.png[Creating a new note from the flyout, 80%]
+.. In the event details flyout, expand the flyout, then go the **Notes** tab in the left panel.
+. Enter a note into the text box, then click **Add note**. 
++
+Notes that you add to alerts display in the right panel of the alert details flyout. Notes that you add to events display in the **Notes** tab.
++
+[role="screenshot"]
+image::images/new-note-added-flyout.png[New note added to an alert]
+
+[discrete]
+[[add-notes-timelines]]
+== Add notes to Timelines
+
+IMPORTANT: You can only add notes to saved Timelines.  
+
+There are two ways to add notes to a saved and open Timeline: use the **Notes** Timeline tab, or use the details flyout for alerts and events that you're investigating in Timeline. 
+
+* From the **Notes** Timeline tab, you can create a new note that's attached to the entire Timeline. 
+* From the details flyout, any notes that you add to alerts and events are automatically attached to the Timeline you're currently working on. If you want to add a note for the alert or event only, deselect the **Attach to current Timeline** option.
+
+[discrete]
+[[manage-notes]]
+== Find and manage notes 
+
+//Will need to revisit the navigation instructions below. The nav path to the Notes page differs between the Classic nav view (Manage -> Investigations -> Notes) and the Security solution view (Investigations -> Notes)
+
+The **Notes** page allows you to view and interact with all existing notes. To access the page, find **Investigations** in the main menu or look for “Investigations” using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field], then go to **Notes**.
+
+TIP: You can also manage notes for alerts, events, and Timelines from the **Notes** <<expanded-notes-view,tab>> in the details flyout or in Timeline.
+
+[role="screenshot"]
+image::images/notes-management-page.png[Notes management page]
+
+From the Notes page, you can:
+
+* Search for specific notes or filter them by:
+** The user who created the notes
+** The type of object that notes are attached to (notes can be attached to alerts, events, Timelines, or nothing)
```

Comment:
I wonder here if we should omit the `nothing` option, just so that we do not confuse the users. Nothing should never happen, I had only added this option juuuust in case something was wrong on the environment. I feel like we do not have to mention this in the docs, wdyt?

Reply:
Yeah, probably a good idea, especially since this situation should never happen. I'll update the line item. Thanks!

```suggestion
** The type of object that notes are attached to (notes can be attached to alerts, events, or Timelines)
```